pilopt/src/lib.rs

```diff
@@ -22,6 +22,7 @@ pub fn optimize<T: FieldElement>(mut pil_file: Analyzed<T>) -> Analyzed<T> {
     remove_constant_fixed_columns(&mut pil_file);
     simplify_identities(&mut pil_file);
     extract_constant_lookups(&mut pil_file);
+    extract_empty_permutations(&mut pil_file);
```

Comment:
```suggestion
    extract_inactive_permutations(&mut pil_file);
```
(empty for me is `[] is []`).